pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataSetup.java

```diff
@@ -192,7 +192,7 @@ private static class Arguments {
     }
 
     /**
-     * a wrapper for creating a persistent node with store.put but ignore exception of node exists.
+     * a wrapper for creating a persistent node with store. put but ignore exception of node exists.
```

Comment:
"put but...", perhaps the grammar of this comment could be fixed at the same time?

Reply:
makes sense, thanks